format-specs/geoparquet.md

```diff
@@ -50,57 +50,116 @@ Version of the geoparquet spec used, currently 0.3.0
+| ------------- | ------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| crs           | JSON object         | **REQUIRED** [PROJJSON](https://proj.org/specifications/projjson.html) JSON object representing the Coordinate Reference System (CRS) of the geometry. This field must be set to `null` if CRS is not known. |
+| encoding      | string              | **REQUIRED** Name of the geometry encoding format. Currently only 'WKB' is supported.                                                                                                                        |
+| geometry_type | string or \[string] | **REQUIRED** The geometry type(s) of all geometries, or 'Unknown' if they are not known.                                                                                                                     |
+| orientation   | string              | **OPTIONAL** Winding order of exterior ring of polygons. If present must be 'counterclockwise'; interior rings are wound in opposite order. If absent, no assertions are made regarding the winding order.   |
+| edges         | string              | **OPTIONAL** Name of the coordinate system for the edges. Must be one of 'planar' or 'spherical'. The default value is 'planar'.                                                                             |
+| bbox          | \[number]           | **OPTIONAL** Bounding Box of the geometries in the file, formatted according to [RFC 7946, section 5](https://tools.ietf.org/html/rfc7946#section-5).                                                        |
+| epoch         | double              | **OPTIONAL** Coordinate epoch in case of a dynamic CRS, expressed as a decimal year.                                                                                                                         |
 
 #### crs
 
-The Coordinate Reference System (CRS) is an optional parameter for each geometry column defined in geoparquet format.
+The Coordinate Reference System (CRS) is a required parameter for each geometry
+column defined in geoparquet format.
+
+The CRS must be provided in
+[PROJJSON](https://proj.org/specifications/projjson.html) format, which is a
+JSON encoding of
+[WKT2:2019 / ISO-19162:2019](https://docs.opengeospatial.org/is/18-010r7/18-010r7.html),
+which itself implements the model of
+[OGC Topic 2: Referencing by coordinates abstract specification / ISO-19111:2019](http://docs.opengeospatial.org/as/18-005r4/18-005r4.html).
+Apart from the difference of encodings, the semantics is intended to be exactly
+the same as WKT2:2019, and PROJJSON can be morphed losslessly from/into
+WKT2:2019.
```

Comment:
It looks like PROJJSON mentions [a few deviations](https://proj.org/specifications/projjson.html#deviations-with-the-wkt2-2019-specification) from WKT2:2019.  Maybe "exactly" and "losslessly" are too strong here.

How about "Apart from the difference of encodings, the semantics of PROJJSON are intended to match WKT2:2019, and a CRS in one encoding can generally be represented in the other"?

Reply:
Regarding this, it could be wise, for greater interoperability, to discourage the use of BoundCRS objects, softly for top-level (as this maps to standard WKT2), and strongly for BoundCRS embedded in CompoundCRS (as this is a PROJ extension), for the PROJJSON CRS of GeoParquet.
BoundCRS objects are mostly the port of PROJ.4 +nadgrids/+towgs84/+geoidgrids constructs that are less necessary nowadays since PROJ can use its database to infer transformations between datums.
So the list of top-level objects suggested for use in PROJJSON would be:
- GeographicCRS 2D (or 3D if 3D data),
- ProjectedCRS 2D (note: ProjectedCRS 3D with ellipsoidal height is supported by PROJ, but still a bit of esoteric)
- CompoundCRS with horizontal part being a GeographicCRS 2D or a ProjectedCRS 2D and vertical part being a VerticalCRS
- and perhaps, GeodeticCRS with cartesian geocentric coordinate system (X,Y,Z) (probably of marginal use)